VmAgent.Core/VmConfiguration.cs

```diff
@@ -42,6 +42,11 @@ public class VmConfiguration
         // should use `PF_SHARED_CONTENT_FOLDER` instead (which points to wherever the folder is mounted in the container)
         public const string SharedContentFolderVmVariable = "PF_SHARED_CONTENT_FOLDER_VM";
 
+        // Used by the games to share user generated content (and other files that are downloaded once, used multiple times).
+        // Also available to VM-level scripts for special cases (e.g. Watson needs to set the Kernel core dump path to this path
+        // so that container-based servers place their dump files in the path available to the container)
+        private const string SharedContentFolderEnvVariable = "PF_SHARED_CONTENT_FOLDER";
```

Comment:
should we also add some tests to make sure the correct set of env variables are being set?

Reply:
Added unit tests